libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -447,7 +447,9 @@ public void testDuplicateEntitlements() {
     public void testFilesEntitlementsWithExclusive() {
         var baseTestPath = Path.of("/base").toAbsolutePath();
         var testPath1 = Path.of("/base/test").toAbsolutePath();
+        var testPath1String = testPath1.toString();
```

Comment:
I thought these `toString` calls are not needed if the `Strings.format` call is using `%s`?